Update/omake_02.txt

```diff
@@ -1726,7 +1726,7 @@ void main()
 	PlaySE(4, "S20/09/440900118", 128, 64);
 	OutputLine(NULL, "…『鬼隠し』でも、今回の『綿流し』でも。",
 		   NULL, " ...Both last time in Onikakushi, and this time in Watanagashi.", Line_WaitForInput);
-	PlaySE(4, "S02/00/440900118_2", 128, 64);
+	PlaySE(4, "S02/00/440900119", 128, 64);
```

Comment:
In this case 440900118_2 is 100% correct. 440900119 doesn't fit there, the closest to fit is 440900118 but still missing the last chunk.